src/server/qgsfeaturefilter.cpp

```diff
@@ -0,0 +1,42 @@
+/***************************************************************************
+                      qgsfeaturefilterprovidergroup.cpp
```

Comment:
qgsfeaturefilter.cpp

Reply:
Those useless informations in comments really start to annoy me...

Reply:
:)

Reply:
@pvalsecc speaking of which: information doesn't have plural in English - unlike French or German ;)